inventory_management_system_api/core/config.py

```diff
@@ -17,6 +19,34 @@ class APIConfig(BaseModel):
     root_path: str = ""  # (If using a proxy) The path prefix handled by a proxy that is not seen by the app.
 
 
+class AuthenticationConfig(BaseModel):
+    """
+    Configuration model for the JWT access token authentication/authorization.
+    """
+
+    enabled: bool = False
+    public_key_path: Optional[str] = Field(default=None, validate_default=True)
+    jwt_algorithm: Optional[str] = Field(default=None, validate_default=True)
+
+    @field_validator("public_key_path", "jwt_algorithm")
+    @classmethod
+    def validate_unit(cls, field_value: str, info: ValidationInfo) -> Optional[str]:
```

Comment:
This function name should be changed.

Reply:
Oops, I forgot to change this after pasting it.